resources/src/Data.tsx

```diff
@@ -1,4 +1,5 @@
-import * as React from "react";
+import * as React from "react"
```

Comment:
```suggestion
```

Reply:
Yeah it's better to import the required component directly 👍